src/stock-operations/add-stock-operation/stock-items-addition-row.component.tsx

```diff
@@ -273,14 +329,10 @@ const StockItemsAdditionRow: React.FC<StockItemsAdditionRowProps> = ({
                   placeholder={
                     requiresBatchUuid &&
                     !requiresActualBatchInformation &&
-                    row?.stockBatchUuid in batchBalance
+                    batchBalance
                       ? `Bal: ${
-                          batchBalance[
-                            row?.stockBatchUuid
-                          ]?.quantity?.toLocaleString() ?? ""
-                        } ${
-                          batchBalance[row?.stockBatchUuid]?.quantityUoM ?? ""
-                        }`
+                          batchBalance?.quantity?.toLocaleString() ?? ""
+                        } ${batchBalance?.quantityUoM ?? ""}`
```

Comment:
This quantityUoM should be the default selection in the Qty UoM field. In the video, the balance is in TIN but the pre-filled Qty UoM shows are Strip-20. May be we can align that.

Reply:
I think that is a data issue @ojwanganto , but that batch number is returning that as the balance, its the same even in 2.x but you are right I also think its a bug. We need to fix it
<img width="1518" alt="Screenshot 2024-08-09 at 16 35 05" src="https://github.com/user-attachments/assets/469ccff2-04ad-4edd-9422-7dcf9c32e982">

Reply:
@hadijahkyampeire i agree this might be a bug but lets first try and test this with a different item see what i returned for as  the balance

Reply:
https://github.com/user-attachments/assets/aab76779-e8f7-489d-a073-c952458ef42c

Reply:
Yah @jabahum I already get the balance but we need to make some tweaks on how this works especially the UoM, it should just be a prefilled textField as opposed to a select because a batch usually has the unit and quantity left so the Quantity and UoM will depend on that. That's what I am left with to have this ready.

Reply:
alright noted @hadijahkyampeire